### PR TITLE
Move docs readme to separate file

### PR DESCRIPTION
**Pull request by rmoff**
Since github now renders the README of a folder more prominently it makes sense to split this doc out here.

### CODE REVIEW

**Review by adipolak — CHANGES_REQUESTED** [review]
hi, thanks for this!
I added a tiny comment that could help with discoverability for folks who want to contribute to docs.

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# lakeFS Documentation
+
+Any contribution to the docs, whether it is in conjunction with a code contribution or as a standalone, is appreciated.
+
+Please see [the contributing guide](contributing.md) for details on contributing to lakeFS in general. 
+
+📝  Notice! lakeFS documentation is written using Markdown. make sure to familiarize yourself with the [Markdown Guide](https://www.markdownguide.org/basic-syntax/#heading-best-practices).
```
Comment (arielshaqed) [inline docs/README.md:7]: Nit:
```suggestion
📝  Notice! lakeFS documentation is written using Markdown.  Make sure to familiarize yourself with the [Markdown Guide](https://www.markdownguide.org/basic-syntax/#heading-best-practices).
```

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+Customizing the lakeFS docs site should follow the following guidelines: [Just The Docs Customization](https://just-the-docs.github.io/just-the-docs/docs/customization/) and style-guide.
+
+### lakeFS Style Guide:
+* Don't use unnecessary tech jargon or vague/wordy constructions - keep it friendly, not condescending.
+* Be inclusive and welcoming - use gender-neutral words and pronouns when talking about abstract people like developers).
```
Comment (arielshaqed) [inline docs/README.md:13]: Nit:
```suggestion
* Be inclusive and welcoming - use gender-neutral words and pronouns when talking about abstract people like users and developers.
```

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+* Replace complex expressions with simpler ones.
+* Keep it short - 25-30 words max per sentence.  Otherwise, your readers might get lost on the way. 
+* Use active voice instead of passive. For example: This feature can be used to do task X. vs. You can use this feature to do task X. The second one reads much better, right?
+* You can explain things better by including examples. Show, not tell. Use illustrations, images, gifs, code snippets, etc.
+* Establish a visual hierarchy to help people quickly find the information they need. Use text formatting to create levels of title and subtitle (such as h1 to h6 headings in HTML).
```
Comment (arielshaqed) [inline docs/README.md:18]: *Suggestion:*
```suggestion
* Establish a visual hierarchy to help people quickly find the information they need. Use text formatting to create levels of title and subtitle (such as `#` to `######` markdown headings).  The title of every page should use the topmost heading `#`; all other headings on the page should use lower headers `##` to `######`.
```

File: docs/README.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+### Test your changes locally
+
+To render the documentation locally and preview changes you can run the Jeykll server under Docker.
```
Comment (arielshaqed) [inline docs/README.md:22]: It should also be possible to `make docs-serve`, assuming you have the relevant tooling.  Do we want to mention that option?

File: docs/README.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+   docker run --rm \
+              --name lakefs_docs \
+              -e TZ="Etc/UTC" \
+              --publish 4000:4000 --publish 35729:35729 \
+              --volume="$PWD/docs:/srv/jekyll:Z" \
+              --volume="$PWD/docs/.jekyll-bundle-cache:/usr/local/bundle:Z" \
+              --interactive --tty \
+              jekyll/jekyll:3.8 \
+              jekyll serve --livereload
+   ```
```
Comment (arielshaqed) [inline docs/README.md:36]: Consider making this a Makefile target.

**Review by arielshaqed — COMMENTED** [review]
Neat, THANKS!  Like Adi's suggestion about telling people about the docs README.

File: docs/README.md (hunk 5 above)
Comment (rmoff) [reply docs/README.md:36]: as well as, or instead of, the existing `docs-serve` ?

File: docs/README.md (hunk 4 above)
Comment (rmoff) [reply docs/README.md:22]: Good point. I'll mention it.

File: docs/contributing.md [hunk 6]
```diff
@@ -121,98 +121,9 @@ Any contribution to the docs, whether it is in conjunction with a code contribut
 Documentation of features and changes in behaviour should be included in the pull request.
 You can create separate pull requests for documentation changes only.
```
[the remark this answers is not in the document]
Comment (rmoff) [reply docs/contributing.md:123]: good idea - done.

File: docs/README.md (hunk 5 above)
Comment (arielshaqed) [reply docs/README.md:44]: Probably in addition: Makefiles that just run Docker images are a bit of a nuisance for developers.  Perhaps `docs-serve-docker` would be a better name??...?

**Review by arielshaqed — APPROVED** [review]
Neat, thanks!

File: docs/README.md (hunk 5 above)
Comment (rmoff) [reply docs/README.md:44]: Done

**Pull request merged**